### ISSUE TITLE
Denote 'private' and 'public' Hackspaces

### ISSUE THREAD

**Issue by syberphunk** [UKHackspaceFoundation/website#91]
There exist hackspaces and makerspaces which are not on publicly accessible grounds, these are either on private territory (such as an [RAF base](https://www.raf.mod.uk/raf90signals/news/index.cfm?storyid=D9AEF5E3-5056-A318-A8692DB6295B7D1D)) or they're based inside educational establishments (such as at a [university or college](https://www.imperial.ac.uk/advanced-hackspace/)) and are only or mainly available to students.

In the spirit of collaboration, accessibility and awareness, it would be fair for 'spaces such as these to be list-able with a clear message to those viewing the site as to what status they're in.

I propose a simple 'public' vs 'private' type of filter or category.

If the 'space is listed on the site as 'public' or 'private', perhaps it could have a superscript '?' which on mouse-over defined what they meant, or on mouse-over of the public/private text.

**Comment by amunizp** [comment]
If we are exploring this:

I'd rather use *Open* and *Closed* group. Closed groups to be listed in a separate page

Other examples of closed group could be LGBT+ only, only female, or mens In sheds?

**Comment by syberphunk** [comment]
No.

The reason I am against 'open' and 'closed' is because those words have a distinct meaning and that represents whether or not the location is explicitly accessible via 'opening hours' or 'opening times'.

Adding additional definition to this request confuses matters, especially with the definitions you're suggesting, especially if any Hackspace goes down the route of SpaceAPI implementation where it particularly uses the wording 'open' and 'closed' to denote this and we put that on the website.

Please, I would prefer it if you took your suggestion to discourse or telegram to lay out the details of how your suggestion could be applied and keep this issue to discussing the ability to implement the enhancement I have proposed.

**Comment by castaway** [comment]
@syberphunk Public versus Private (or Restricted?) seem like better terms to me

**Comment by syberphunk** [comment]
Agreed, Public / Restricted sounds better. As restrictions can apply to accessibility, location, or other requirements.

Not sure Private would be best, eg. if I joined the University or RAF I would then have lifted the restriction and be able to join the 'space.
